packages/runtime/src/controls/rich-text/rich-text.ts

```diff
@@ -70,7 +70,7 @@ export type RichTextControlMessage =
 export class RichTextControl extends PropController<RichTextControlMessage> {
   private editor: Editor | null = null
 
-  recv(message: RichTextControlMessage): void {
+  recv(message: PropControllerMessage): void {
```

Comment:
Why would the rich text control receive a message that's not a rich text control message?

Reply:
See my other comment on why.